lib/selector.js

```diff
@@ -57,8 +59,35 @@ class Selector extends Prefixer {
     }
 
     let prefixeds = {}
-    for (let prefix of this.possible()) {
-      prefixeds[prefix] = this.replace(rule.selector, prefix)
+
+    if (rule.selector.includes(',')) {
+      let separatedRule = list.comma(rule.selector)
+
+      this.all.add.selectors.forEach(selector => {
+        let name = selector.name
+        let selToProc = separatedRule.filter(el => el.includes(name))
+
+        if (selToProc.length) {
+          let possible = selector.possible()
+          prefixeds[name] = {}
+
+          possible.forEach(prefix => {
+            if (selToProc.length > 1) {
```

Comment:
I think we can remove this `if`. Array with a single element will have the same result after processing with `map` and `join`. It will be a little slower, but it is not a problem here since the main idea is to reduce code complexity.

Reply:
Removed it.